bin/run_automated_checks

```diff
@@ -3,4 +3,4 @@
 set -e
 
 # all checks get run if none are specified
-bundle exec bin/run_checks.rb
+bundle exec bin/run_checks.rb $@
```

Comment:
See http://ss64.com/bash/syntax-parameters.html
Basically it expands to 'all arguments given in the current scope'
But see also http://ss64.com/bash/shift.html